src/components/Pagination/Pagination.tsx

```diff
@@ -2,0 +1,69 @@
+import React from 'react';
+
+type Props = {
+  total: number;
+  perPage: number;
+  currentPage: number;
+  onPageChange: React.Dispatch<React.SetStateAction<number>>;
+};
+
+export const Pagination = ({
+  total,
+  perPage,
+  currentPage,
+  onPageChange,
+}: Props) => {
+  const numOfPages = Math.ceil(total / perPage);
+  const pages = Array.from({ length: numOfPages }, (_, i) => i + 1);
+  const prevHandler = () => currentPage !== 0 && onPageChange(currentPage - 1);
+  const nextHandler = () =>
+    currentPage !== numOfPages - 1 && onPageChange(currentPage + 1);
+
+  return (
+    <>
+      <ul className="pagination">
+        <li className={`page-item ${currentPage === 0 && 'disabled'}`}>
+          <a
+            data-cy="prevLink"
+            className="page-link"
+            href="#prev"
+            aria-disabled={currentPage === 0 ? 'true' : 'false'}
```

Comment:
This is a bug. Your `currentPage` is 1-indexed, but here you are treating it as 0-indexed. You should check if `currentPage` is not equal to 1 instead of 0.